singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -336,8 +336,8 @@ inline SpinerEOSDependsRhoSie::SpinerEOSDependsRhoSie(const std::string &filenam
   status += H5Gclose(lTGroup);
   status += H5Gclose(lEGroup);
   status += H5Gclose(matGroup);
-  status += H5Fclose(file);
   status += H5Gclose(coldGroup);
+  status += H5Fclose(file);
```

Comment:
The file object needs to be closed after the group object.

Reply:
The tests passed because this only  happens after the data has been properly read. It's essentially a failure at the cleanup step. And HDF5 raises a warning, not an error.

We maybe would have caught it if we looked for output to stderr, or if we had an address sanitizer test run on the full suite on re-git.